LongestPole/Main.lean

```diff
@@ -122,9 +122,8 @@ open System in
 def countLOC (modules : List Name) : IO (NameMap Float) := do
   let mut r := {}
   for m in modules do
-    let fp := FilePath.mk ((← findOLean m).toString.replace ".lake/build/lib/" "")
-      |>.withExtension "lean"
-    if ← fp.pathExists then
+    if let .some fp ← Lean.SearchPath.findModuleWithExt [s!".{FilePath.pathSeparator}"] "lean" m
```

Comment:
```suggestion
    if let .some fp ← Lean.SearchPath.findModuleWithExt [".".push FilePath.pathSeparator] "lean" m
```
this may also work, although I am not sure if there is any substantial difference.

Reply:
(I personally find this version more readable.)